packages/nextjs/src/index.server.ts

```diff
@@ -124,9 +131,16 @@ function addServerIntegrations(options: NextjsOptions): void {
   options.integrations = integrations;
 }
 
+// TODO (v8): Remove this
+/**
+ * @deprecated Use the constant `IS_BUILD` instead.
+ */
+const deprecatedIsBuild = (): boolean => isBuild();
+// eslint-disable-next-line deprecation/deprecation
+export { deprecatedIsBuild as isBuild };
```

Comment:
l: ehhh do we have to deprecate this? Is anyone even using this export?

Reply:
I have no idea, but it seems to me we don't lose much by deprecating it.